src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -173,16 +173,33 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
 
             href: {
                 condition: el => {
+                    if (LocationAccessorsInstrumentation.isLocationWrapper(el))
```

Comment:
This changes the order of checks. In the beginning have to go most likely and fast checks.